File: 01_Language/01_Functions/go/src/functions/ctype_graph.go
package functions

func CtypeGraph(s string) bool {
	if len(s) == 0 {
		return false
	}
	for i := 0; i < len(s); i++ {
		codePoint := int(s[i])
		if codePoint > 32 && codePoint <= 126 {
			continue
		}
		return false
	}
	return true
}

File: 01_Language/01_Functions/js/ctype_graph.js
function ctype_graph(text) {
    if (text === null || text === undefined || text === '') return false
    for (let i in text) {
        let codePoint = text.charCodeAt(i)
        if (codePoint > 32 && codePoint <= 126) {
            continue
        }
        return false
    }
    return true
}
